src/TSTransformer/macros/propertyCallMacros.ts

```diff
@@ -6,6 +6,7 @@ import { ensureTransformOrder } from "TSTransformer/util/ensureTransformOrder";
 import { isUsedAsStatement } from "TSTransformer/util/isUsedAsStatement";
 import { offset } from "TSTransformer/util/offset";
 import { assert } from "Shared/util/assert";
+import { SyntaxKind } from "typescript";
```

Comment:
Don't import anything from typescript directly.

Reply:
If you need it, use `import ts from "byots"`

Reply:
Wait howd that get added I never touched the imports